src/components/transactions/TxDetails/Summary/index.tsx

```diff
@@ -73,12 +75,15 @@ const Summary = ({ txDetails, defaultExpanded = false }: Props): ReactElement =>
 export default Summary
 
 export const PartialSummary = ({ safeTx }: { safeTx: SafeTransaction }) => {
+  const { safe } = useSafeInfo()
   const txData = safeTx.data
   return (
     <>
-      <TxDataRow title="safeTxGas:">
-        <SafeTxGasForm />
-      </TxDataRow>
+      {safe.version && isLegacyVersion(safe.version) && (
```

Comment:
Don’t you also need to check if it’s 0 for 1.3.0+ Safes? For when a safe app sends a custom safeTxGas

Reply:
Good point, then we should show it.

However, if the user was then to set it to `0`, the field would disappear if it's rendered relative to `0`. Should we just _always_ show it?

Reply:
Well, bro, that's how it was :D

Reply:
Seriously though, I think this whole Advanced details section is quite obscure and it's probably OK to just always show it there. In fact, that's how it has always been for _existing_ transactions. Now it's just also editable for new txs.
Maybe we should add some info tooltip discouraging folks from touching it if not necessary.